src/webapi/history.rs

```diff
@@ -0,0 +1,91 @@
+#![cfg(feature = "serde")]
```

Comment:
Looks like you forgot to remove this? (:

Reply:
Oh, indeed, my bad.